Userland/Libraries/LibWasm/Tests/Parser/spec-testsuite.js

```diff
@@ -0,0 +1,23 @@
+let haveSpecTestSuite = false;
+try {
+    readBinaryWasmFile("Fixtures/SpecTests/address.wasm");
+    haveSpecTestSuite = true;
+} catch {}
+
+let test_function = haveSpecTestSuite ? test : test.skip;
```

Comment:
Please don't mix pascalCase and snake_case here (or anywhere) - let's stick with pascalCase in JS files.

Reply:
Sure. my brain must've been in C++ mode.
Can we make prettier warn about this?

Reply:
would be nice, but I don't think we can. ESLint definitely could, but that seems overkill lol